### PR TITLE
Avoid ValueError for TFDS with BBox on the last pixel

### LINKED ISSUE #1208: Failed to build TFDS with openimages dataset
There is a subset of openimages (face). See: https://docs.blue-oil.org/tutorial/image_det.html#preparation
I used this dataset and run build_tfds command to build TFDS, but I got the following error.

```
    'BBox coordinates should be between 0 and 1. Got {}.'.format(bbox))
ValueError: BBox coordinates should be between 0 and 1. Got BBox(ymin=0.0, xmin=0.108385, ymax=0.885862, xmax=1.0000000000000002).
```

### PR DESCRIPTION
**Pull request by hadusam**
## What this patch does to fix the issue.
Fix #1208

### CODE REVIEW

File: blueoil/utils/tfds_builders/object_detection.py [hunk 1]
```diff
@@ -77,8 +81,8 @@ def _generate_examples(self, dataset):
                     "bbox": tfds.features.BBox(
                         ymin / height,
                         xmin / width,
-                        (ymin + h) / height,
-                        (xmin + w) / width,
+                        _clip_max((ymin + h) / height),
```
Comment (iizukak) [inline blueoil/utils/tfds_builders/object_detection.py:84]: @hadusam 
`min((ymin + h) / height, 1.0)` is not enough for you?

Comment (hadusam) [reply blueoil/utils/tfds_builders/object_detection.py]: @iizukak 
Yeah, it's enough! Thanks!!!
Fixed.

**Review by iizukak — APPROVED** [review]
OA

**Review by tfujiwar — APPROVED** [review]
RA

**Pull request merged**